MonoGame.Framework/Audio/SoundEffectInstance.XAudio.cs

```diff
@@ -60,6 +60,86 @@ private void PlatformApply3D(AudioListener listener, AudioEmitter emitter)
+            //
+            // From MSDN:
+            //  The XNA Framework uses a right-handed coordinate system, 
+            //  with the positive z-axis pointing toward the observer when the positive x-axis is pointing to the right, 
+            //  and the positive y-axis is pointing up. 
+            //
+            // Programmer Notes:         
+            //  According to this description the z-axis (forward vector) is inverted between these two coordinate systems.
+            //  Therefore, we need to negate the z component of any position/velocity values, and negate any forward vectors.
+
+            forward *= -1.0f;
+            pos.Z *= -1.0f;
+            vel.Z *= -1.0f;
+
+            if (_dxEmitter == null)
+                _dxEmitter = new Emitter();
+
+            _dxEmitter.Position = new SharpDX.Vector3(pos.X, pos.Y, pos.Z);
+            _dxEmitter.Velocity = new SharpDX.Vector3(vel.X, vel.Y, vel.Z);
+            _dxEmitter.OrientFront = new SharpDX.Vector3(forward.X, forward.Y, forward.Z);
+            _dxEmitter.OrientTop = new SharpDX.Vector3(up.X, up.Y, up.Z);
+            _dxEmitter.DopplerScaler = emitter.DopplerScale;
+
+            return _dxEmitter;
+        }
+
+        private SharpDX.X3DAudio.Listener ToDXListener(AudioListener listener)
+        {
+            // Pulling out Vector properties for efficiency.
+            var pos = listener.Position;
+            var vel = listener.Velocity;
+            var forward = listener.Forward;
+            var up = listener.Up;
+
+            // From MSDN:
+            //  X3DAudio uses a left-handed Cartesian coordinate system, 
+            //  with values on the x-axis increasing from left to right, on the y-axis from bottom to top, 
+            //  and on the z-axis from near to far. 
+            //  Azimuths are measured clockwise from a given reference direction. 
+            //
+            // From MSDN:
+            //  The XNA Framework uses a right-handed coordinate system, 
+            //  with the positive z-axis pointing toward the observer when the positive x-axis is pointing to the right, 
+            //  and the positive y-axis is pointing up. 
+            //
+            // Programmer Notes:         
+            //  According to this description the z-axis (forward vector) is inverted between these two coordinate systems.
+            //  Therefore, we need to negate the z component of any position/velocity values, and negate any forward vectors.
+
+            forward *= -1.0f;
+            pos.Z *= -1.0f;
+            vel.Z *= -1.0f;
+
+            if (_dxListener == null)
+                _dxListener = new Listener();
+
+            _dxListener.Position = new SharpDX.Vector3(pos.X, pos.Y, pos.Z);
+            _dxListener.Velocity = new SharpDX.Vector3(vel.X, vel.Y, vel.Z);
+            _dxListener.OrientFront = new SharpDX.Vector3(forward.X, forward.Y, forward.Z);
+            _dxListener.OrientTop = new SharpDX.Vector3(up.X, up.Y, up.Z);
```

Comment:
Same here....  see the original code:

https://github.com/mono/MonoGame/blob/develop/MonoGame.Framework/Windows8/SharpDXHelper.cs#L227